Dockerfile

```diff
@@ -1,7 +1,38 @@
-FROM python:3.9-bullseye
+FROM ubuntu:22.04
 
-# Remove after direct git reference to Prophet is removed.
-RUN apt-get update && apt-get -y install git
+ENV DEBIAN_FRONTEND=noninteractive
+ENV TZ=Etc/UTC
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+
+# install dependencies
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get remove -y python3.10 && \
+    apt-get install -y --no-install-recommends software-properties-common gnupg2 && \
+    add-apt-repository -y ppa:deadsnakes/ppa && \
+    apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y --no-install-recommends \
+    build-essential curl \
+    libpq-dev \
+    libsasl2-dev \
+    libssl-dev libffi-dev \
+    python3.9 python3.9-dev python3.9-venv libpython3.9-dev libpython3.9 \
+    python3.9-distutils \
+    unixodbc-dev git && \
+    apt-get clean -qq -y && \
+    apt-get autoclean -qq -y && \
+    apt-get autoremove -qq -y && \
+    rm -rf /var/lib/apt/lists/*
+RUN ln -s /usr/bin/python3.9 /usr/bin/python && \
+    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
+    python3.9 get-pip.py
+
+# extra dependencies for soda-scientific
```

Comment:
shouldnt these be in setup.py of scientific package?

Reply:
They may be but for some weird unknown reason sometimes installing them independently leads to better success. Maybe @jonas has a more scientific answer since he did it recently (see what I did there? Scientific? Ok...)

Reply:
When they were added through setup.py, it failed during testing, so I had to install them upfront through the Docker image instead.